import cn.hutool.core.exceptions.ExceptionUtil;
public class FunctionUse1620 {
public void funcUse() {
ExceptionUtil exceptionutil = new ExceptionUtil();
exceptionutil.getRootCauseMessage(null);
}
}